SPECS/clang/clang.spec

```diff
@@ -73,7 +73,6 @@ make clang-check
 %clean
```

Comment:
Nit: `%clean` sections that just remove the buildroot can be removed. Our version of RPM adds this automatically (same for lines at the beginning of `%install` sections that wipe out the buildroot)